java/arcs/core/data/expression/Stringifier.kt

```diff
@@ -79,6 +79,11 @@ class ExpressionStringifier(val parameterScope: Expression.Scope = ParameterScop
                 argExpr.accept(this)
             }
         )
+
+    override fun <T> visit(expr: Expression.OrderByExpression<T>): String =
+        expr.qualifier.accept(this) + "\norder by " + expr.selectors.joinToString(
```

Comment:
no space between order and by. We should start a separate file with tests for the stringifier and json serializer.

Reply:
Done